src/Behat/Gherkin/Node/BackgroundNode.php

```diff
@@ -77,7 +77,7 @@ public function getTitle()
      */
     public function hasSteps()
     {
-        return 0 < count($this->steps);
+        return count($this->steps) > 0;
```

Comment:
`empty` is quite problematic due to all its magic behavior (if you have a native type enforcing that the value is an array, the behavior is quite sane. But never use it on something that can be a string). phpstan-strict-rules even has a rule to forbid using `empty`. I would not encourage using it.

@acoulton that's something which is a lot easier to check with static analysis than with a CS fixer though, so not sure there is such fixer.

Reply:
I agree - definitely avoid `empty` wherever possible as it can give very unexpected behaviour, for example `empty($this->steps)` will silently return false rather than reporting any sort of error if there is no `->steps` property defined on the class, because it internally counts as an `isset()` call.

Good point that it'll be hard to be sure of the safety of this without strict types and static analysis, so probably best to stick with `count() === 0` for now.

Reply:
> Good point that it'll be hard to be sure of the safety of this without strict types and static analysis, so probably best to stick with count() === 0 for now.

OK, I'll revert it then.